### PR TITLE
Update debian setup script to match GCI.  This is similar to pr #57051.

### PR DESCRIPTION
**Pull request by dnardo**

### CODE REVIEW

**Comment by dnardo** [comment]
Running tests now I'll update when I'm sure things are working fine.

**Review by MrHohn — APPROVED** [review]
/lgtm

**Comment by MrHohn** [comment]
Verified this is working as expected.

/lgtm

**Comment by dnardo** [comment]
Ok this looks good now when I setup a test cluster

NAME                                             READY     STATUS    RESTARTS   AGE
etcd-empty-dir-cleanup-e2e-test-dnardo-master    1/1       Running   0          7m
etcd-server-e2e-test-dnardo-master               1/1       Running   0          7m
etcd-server-events-e2e-test-dnardo-master        1/1       Running   0          7m
event-exporter-v0.1.7-91598863-18ql0             2/2       Running   0          7m
fluentd-gcp-v2.0.9-9psml                         2/2       Running   0          7m
fluentd-gcp-v2.0.9-knth5                         2/2       Running   0          7m
fluentd-gcp-v2.0.9-q92d8                         2/2       Running   0          7m
fluentd-gcp-v2.0.9-vj9qr                         2/2       Running   0          7m
heapster-v1.4.3-3064730318-hg4v5                 4/4       Running   0          4m
kube-addon-manager-e2e-test-dnardo-master        1/1       Running   0          6m
kube-apiserver-e2e-test-dnardo-master            1/1       Running   0          7m
kube-controller-manager-e2e-test-dnardo-master   1/1       Running   0          7m
kube-dns-3468831164-dqt2b                        3/3       Running   0          6m
kube-dns-3468831164-xfcqc                        3/3       Running   0          7m
kube-dns-autoscaler-244676396-m198z              1/1       Running   0          7m
kube-proxy-e2e-test-dnardo-minion-group-fpdp     1/1       Running   0          7m
kube-proxy-e2e-test-dnardo-minion-group-lwg0     1/1       Running   0          7m
kube-proxy-e2e-test-dnardo-minion-group-t027     1/1       Running   0          7m
kube-scheduler-e2e-test-dnardo-master            1/1       Running   0          7m
kubernetes-dashboard-1265873680-kwn7v            1/1       Running   0          7m
l7-default-backend-3623108927-jb8qm              1/1       Running   0          7m
l7-lb-controller-v0.9.6-e2e-test-dnardo-master   1/1       Running   0          7m
monitoring-influxdb-grafana-v4-wsf5t             2/2       Running   0          7m
npd-v0.4.1-1wbmf                                 1/1       Running   0          7m
npd-v0.4.1-6zhh9                                 1/1       Running   0          7m
npd-v0.4.1-h1pnr                                 1/1       Running   0          7m
npd-v0.4.1-vhnp8                                 1/1       Running   0          7m
rescheduler-v0.3.1-e2e-test-dnardo-master        1/1       Running   0          7m

File: cluster/gce/configure-vm.sh [hunk 1]
```diff
@@ -90,6 +90,38 @@ ensure-local-disks() {
...
+  sysctl -w net.ipv4.conf.all.route_localnet=1
+
+  # We need to add rules to accept all TCP/UDP/ICMP packets.
+  if iptables -L INPUT | grep "Chain INPUT (policy DROP)" > /dev/null; then
+    echo "Add rules to accept all inbound TCP/UDP/ICMP packets"
+    iptables -A INPUT -p TCP -j ACCEPT
+    iptables -A INPUT -p UDP -j ACCEPT
+    iptables -A INPUT -p ICMP -j ACCEPT
+  fi
+  if iptables -L FORWARD | grep "Chain FORWARD (policy DROP)" > /dev/null; then
```
Comment (bowei) [inline cluster/gce/configure-vm.sh]: you can do `grep -q` to not print anything

File: cluster/gce/configure-vm.sh [hunk 2]
```diff
@@ -90,6 +90,38 @@ ensure-local-disks() {
 function config-ip-firewall {
   echo "Configuring IP firewall rules"
 
+  # Do not consider loopback addresses as martian source or destination while
+  # routing. This enables the use of 127/8 for local routing purposes.
+  sysctl -w net.ipv4.conf.all.route_localnet=1
+
+  # We need to add rules to accept all TCP/UDP/ICMP packets.
+  if iptables -L INPUT | grep "Chain INPUT (policy DROP)" > /dev/null; then
```
Comment (bowei) [inline cluster/gce/configure-vm.sh]: `grep -q`

iptables -n -L?

**Comment by dnardo** [comment]
@bowei  I hear ya, but I didn't write the original, and at this point I'm just trying to make things consistent. I would hate to have this be a one off from the others in the GCI directory.   We might want to follow up and clean all  of the scripts up in one shot?

**Comment by wojtek-t** [comment]
I'm merging this manually to see if that fixes the problems in 1.7 branch.

**Comment by wojtek-t** [comment]
FYI - yes, it helped. Thanks for fixing.

**Pull request merged**